src/Weather/Skysight/Metrics.hpp

```diff
@@ -24,6 +24,71 @@ Copyright_License {
 #ifndef WEATHER_SKYSIGHT_METRICS_HPP
 #define WEATHER_SKYSIGHT_METRICS_HPP
 
+#include "Util/tstring.hpp"
+#include "Time/BrokenDateTime.hpp"
 #include <map>
 
+struct LegendColor {
+  unsigned char Red;
+  unsigned char Green;
+  unsigned char Blue;
+};
+
+struct SkysightMetric {
+  const tstring id;
+  const tstring name;
+  const tstring desc;
+  uint64_t last_update = 0;
+  std::map<float, LegendColor> legend;
+
+public:
+  SkysightMetric(tstring _id, tstring _name, tstring _desc):
+    id(_id), name(_name), desc(_desc) {}
+  SkysightMetric(const SkysightMetric &m):
+    id(m.id), name(m.name), desc(m.desc), last_update(m.last_update),
+    legend(m.legend) {}
+};
+
+struct SkysightActiveMetric {
```

Comment:
Documentation missing

Reply:
/*
 * Skysight chart which is overlaid
 */

Reply:
I have no idea what this means.